debug/README.md

```diff
@@ -0,0 +1,40 @@
+# gNOI `Debug` Streaming RPC Design
+
+**Contributors**: hines@google.com, robjs@google.com
+**Last Updated**: 2023-11-04
+
+## Background
+
+* [gNOI Repository](https://github.com/openconfig/gnoi)
+* [gNOI `Debug` service](https://github.com/openconfig/gnoi/tree/master/debug)
+
+For all legacy devices that provided a CLI on box, providers  have leveraged this CLI through services to provide users with the ability to interact with device CLI's to both get information as well as set operational state on devices.  This interaction is very vendor specific and requires significant overhead to maintain the vendor specific bindings throughout the operational life cycle of a device.
```

Comment:
This doesn't parse that well. Suggest:

"Traditional network devices provide a CLI, which providers have leveraged for configuration and retrieval of operational state. The implementation of these CLIs is vendor-specific, and requires specific per-vendor client code and logic. Programmatic use of the CLI is fragile -- being based on parsing unstructured strings designed for human consumption, rather than structured machine-friendly data."